Лабораторная работа 3/task1.py

```diff
@@ -0,0 +1,60 @@
+class Book:
+    """ Базовый класс книги. """
+    def __init__(self, name: str, author: str):
+        self.name = name
+        self.author = author
+
+    def name(self):
+        return self.name
+
+    def author(self):
+        return self.author
+
+    def __str__(self):
+        return f"Книга {self.name}. Автор {self.author}"
+
+    def __repr__(self):
+        return f"{self.__class__.__name__}(name={self.name!r}, author={self.author!r})"
+
+
+class PaperBook:
```

Comment:
Класс PaperBook не наследуется от класса Book
```suggestion
class PaperBook(Book):
```